test/build.info

```diff
@@ -548,7 +548,7 @@ IF[{- !$disabled{tests} -}]
       PROGRAMS{noinst}=sm4_internal_test
     ENDIF
     IF[{- !$disabled{ec} -}]
-      PROGRAMS{noinst}=ec_internal_test curve448_internal_test
+      PROGRAMS{noinst}=ectest ec_internal_test curve448_internal_test
```

Comment:
This change means that ectest is no longer built in the event of "no-ec". This will require an amendment to test/recipes/15-test_ec.t to not attempt to call ectest in the event of a no-ec build. In fact that recipe can be simplified to remove all the various `disabled("ec")` tests with just one that skips everything in a no-ec build.

Reply:
Done.